src/components/TodoList/TodoList.tsx

```diff
@@ -1,6 +1,17 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
+type Props = {
+  todos: Todo[];
+  todoId?: number;
```

Comment:
```suggestion
  selectedTodoId?: number;
```
it will be more descriptive